### PR TITLE
Explicit SwiftLint Phase

### PR DESCRIPTION
**Pull request by xavierLowmiller**
## Description

Uses the ["official" SwiftLint action][action] that posts updates on pull requests.

Screenshot 2020-01-12 at 15 18 24

Screenshot 2020-01-12 at 15 23 07

This action runs on a Linux Runner.

[action]: https://github.com/norio-nomura/action-swiftlint

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -43,16 +43,10 @@ jobs:
...
     steps:
-    - name: Checkout
-      uses: actions/checkout@v1
-
-    - name: Install CocoaPods Dependencies
-      run: pod install --repo-update
-
-    - name: Run Swiftlint
-      run: Pods/SwiftLint/swiftlint lint --config .swiftlint.yml
+    - uses: actions/checkout@v1
```
Comment (fbernutz) [inline .github/workflows/main.yml]: Could you add the `name: Checkout` for this? So, it's consistent with the others.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -43,16 +43,10 @@ jobs:
...
+    - uses: norio-nomura/action-swiftlint@3.0.1
```
Comment (fbernutz) [inline .github/workflows/main.yml]: The same here, could you please add `name: Run SwiftLint Checks` or something? It's way more readable than the default `Run norio-nomura/action-swiftlint@3.0.1`

**Comment by xavierLowmiller** [comment]
> Do you know how this behaves when the job runs on a normal push on master? There is nothing the tool can comment on, so is the job simply red?

Yes. You'll receive a mail with a link to the failed job, but there's no comment as far as I can tell.

**Comment by fbernutz** [comment]
Sounds good! Thanks for the contribution

**Pull request merged**